app/async_consumer.py

```diff
@@ -51,12 +51,12 @@ def connect(self):
             self._url = settings.RABBIT_URLS[server_choice]
 
             try:
-                logger.info('Connecting', amqp_url=self._url, attempt=count)
+                logger.info('Connecting', attempt=count)
```

Comment:
Connecting to what though? Should at least say "Connecting to queue server" or similar without revealing actual rabbit details.